src/modules/auth/auth.controller.ts

```diff
@@ -40,7 +48,19 @@ export class AuthController {
   @ApiInternalServerErrorResponse({
     description: 'Internal server error',
   })
-  async login(@Request() { user }: AuthReqDto): Promise<AccesResponseDto> {
+  async login(@Req() { user }: AuthReqDto): Promise<AccesResponseDto> {
     return this.authService.login(user);
   }
+
+  @Post(ERouteName.SEND_OTP)
+  @ApiOperation({ summary: 'Send otp code to provided email' })
+  async restorePassword(@Body() dto: SendOtpRequestDto): Promise<void> {
+    return await this.otpService.sendOtp(dto.email);
+  }
+
+  @Post(ERouteName.VERIFY_OTP)
+  @ApiOperation({ summary: 'Verify provided otp code' })
+  async verifyOtp(@Body() dto: VerifyOtpRequestDto): Promise<UserEntity> {
```

Comment:
we probably need to use UserDto for output type here. So that it is visible for swagger